driven/storage/adapter.go

```diff
@@ -2322,6 +2322,30 @@ func (sa *Adapter) DeleteAccountRoles(context TransactionContext, accountID stri
 	return nil
 }
 
+// DeleteAccountOrgAppsMemberships deletes account org_apps_membership
+func (sa *Adapter) DeleteAccountOrgAppsMemberships(context TransactionContext, accountID string, orgApps []model.OrgAppMembership) error {
```

Comment:
This function also has a wrong name. As I see it does not delete but update the memberships. You should use _$pull_ operator to remove only the desired ones.